### PR TITLE
Sniffles

### PR DESCRIPTION
**Pull request by yuukiiwa**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by nvnieuwk — CHANGES_REQUESTED** [review]

File: tests/modules/nf-core/sniffles/main.nf
[the remark this answers is not in the document]
Comment (yuukiiwa) [reply tests/modules/nf-core/sniffles/main.nf:1]: Thanks! Got that added

File: modules/nf-core/sniffles/meta.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        e.g. [ id:'test' ]
+  - bam:
+      type: file
+      description: BAM file
+      pattern: "*.bam"
+  - bai:
+      type: file
+      description: Index of BAM file
+      pattern: "*.bai"
+  - fasta:
```
[the remark this answers is not in the document]
Comment (yuukiiwa) [reply modules/nf-core/sniffles/meta.yml:28]: added it in

File: tests/modules/nf-core/sniffles/main.nf [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SNIFFLES } from '../../../../modules/nf-core/sniffles/main.nf'
+
+workflow test_sniffles {
```
[the remark this answers is not in the document]
Comment (yuukiiwa) [reply tests/modules/nf-core/sniffles/main.nf:7]: Thanks for the suggestion! I tried adding CRAM files locally for testing, but the test failed (please see screenshot attached below). I guess I won't be adding it then
Screenshot from 2023-03-22 14-45-39

[the remark this answers is not in the document]
Comment (nvnieuwk) [reply tests/modules/nf-core/sniffles/main.nf:7]: Something seems wrong with the reference causing Sniffles to not be able to read the CRAM file...

**Review by nvnieuwk — APPROVED** [review]

**Pull request merged**